pkgs/development/python-modules/arviz/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, emcee
+, h5py
```

Comment:
Not used anywhere.

Reply:
Ah, sorry.  It was another test package that I wound up not using.  I'll comment it out and add it to the list.